beetsplug/play.py

```diff
@@ -125,7 +130,7 @@ def _play_command(self, lib, opts, args):
 
         # Check if the selection exceeds configured threshold. If True,
         # cancel, otherwise proceed with play command.
-        if not self._exceeds_threshold(selection, command_str, open_args,
+        if opts.force or not self._exceeds_threshold(selection, command_str, open_args,
```

Comment:
It looks like this is longer than 80 characters now. Do you think you could wrap the `self._exceeds_threshold` call so that it's back under 80 (and to appease the Travis CI gods)?

Thanks!